ctapipe/instrument/camera.py

```diff
@@ -407,7 +407,10 @@ def neighbor_matrix_where(self):
         -------
         ndarray
         """
-        return np.ascontiguousarray(np.array(np.where(self.neighbor_matrix)).T)
+        return np.column_stack([
```

Comment:
You could store `self.neighbor_matrix_sparse.tocoo()` in a temporary variable instead of calculating twice